src/ui/server/doc/openapi_v1.json

```diff
@@ -75,17 +75,17 @@
         }
       }
     },
-    "/filterconfig": {
+    "/query-options": {
       "get": {
-        "operationId": "getFilterConfig",
+        "operationId": "getQueryOptions",
         "tags": [
           "Util"
         ],
-        "summary": "Retrieve information about filter",
-        "description": "Returns filterconfig as JSON.",
+        "summary": "Retrieve the query options",
+        "description": "Returns queryOptions as JSON.",
```

Comment:
In general i would suggest to make the descriptions longer. E.g. here it could be something like:
"Returns valid query options for the isolate endpoints. These queries can be formed by multiple key=value pairs."
This could be more detailed by describing how multiple values are queried for the same key or how the semantics are when querying for multiple values of attributes with type array (union or intersection logic).
The descriptions of the isolate endpoints can than refer to this description.